### ISSUE TITLE
No video with supported format and MIME type found

### ISSUE THREAD

**Issue by EBaalhuis** [nathaneastwood/fablore#2]
Im seeing this error on some hero pages, e.g. Rhinar, Katsu.

On other hero pages the video seems to play correctly, e.g. Fai, Dromai.

I am using Firefox 105.0.3 on PC.

![image](https://user-images.githubusercontent.com/14149179/195612122-d8b68d58-4db7-46f7-bd8e-817f1225f47e.png)

**Comment by nathaneastwood** [comment]
I see it too on iOS Safari. I'll look into it, thanks.

**Comment by nathaneastwood** [comment]
Turns out this wasn't my fault. I store the media via `git lfs` and use GitHub to host the files. I'm not able to access these files myself. I get 1GB free storage. The problem is when I change the files, the old versions remain on GitHub's server - even if I remove them from the git history (I lost my history because I cleared it trying to free up space). 

GitHub attempted to remove these files for me but they got rid of some they shouldn't. They've managed to restore them so the media on the site should be working again...but I'm back to square one for storage constraints. Hopefully they can properly fix it or I'll have to look into a CDN or something to host the media.
